### DISCUSSION TITLE
AutoTracking: Amcrest IP8M-2779EW-AI

### DISCUSSION THREAD

**Discussion by gadgetb0y** [blakeblackshear/frigate#15349] [Ask A Question] [answered]
Based on the [ONVIF PTZ camera recommendations](https://docs.frigate.video/configuration/cameras#onvif-ptz-camera-recommendations), I picked up 6 Amcrest IP8M-2779EW-AI cameras for $80 each on Black Friday. 

Can anyone confirm that PTZ Controls and Autotracking work on this specific model?

Thanks in advance.

**Comment by hawkeye217** [comment, marked as answer]
I saw you posted this same thing on reddit, so I'll reply here for completeness.

> PTZ controls should work, and I can't explicitly confirm autotracking but other modern Amcrest models have been reported by users as working.

**Reply by gadgetb0y** [reply]
Thanks. I really appreciate the quick response.
